api/src/test/java/com/basistech/rosette/api/RosetteAPITest.java

```diff
@@ -158,19 +151,15 @@ public void testMatchName() throws IOException {
 
     private void verifyNameMatcher(NameSimilarityResponse response) throws IOException {
         NameSimilarityResponse goldResponse = mapper.readValue(responseStr, NameSimilarityResponse.class);
-        assertEquals(goldResponse.getScore(),response.getScore(), 0.0);
+        assertEquals(goldResponse,response);
     }
 
     private NameSimilarityRequest readValueNameMatcher() throws IOException {
-        File input = new File("src/test/mock-data/request", testFilename);
-        return mapper.readValue(input, NameSimilarityRequest.class);
+        return mapper.readValue("{\"name1\": {\"entityType\": \"PERSON\", \"language\": \"eng\", \"script\": \"Latn\", \"text\": \"John Doe\"}, \"name2\": {\"entityType\": \"PERSON\", \"language\": \"fra\", \"script\": \"Latn\", \"text\": \"Jane Doe\"}}", NameSimilarityRequest.class);
```

Comment:
There's no need to test the deserializer here, it was needed because we had to read the object from file. Here you can just construct the object natively in Java.